java/code/src/com/redhat/rhn/domain/action/Action_legacyUser.hbm.xml

```diff
@@ -49,8 +49,7 @@ PUBLIC "-//Hibernate/Hibernate Mapping DTD 3.0//EN"
                         lazy="true" discriminator-value="5">
                         <set name="errata" outer-join="false" table="rhnActionErrataUpdate">
                                 <key column="action_id" />
-                                <many-to-many column="errata_id" outer-join="false"
-                                        class="com.redhat.rhn.domain.errata.Errata" />
+                                <many-to-many column="errata_id" class="com.redhat.rhn.domain.errata.Errata" />
```

Comment:
Are we sure this can just be dropped? I read something about using "lazy=extra" instead.
But I could not really find a good doc about it.

Reply:
I thought it could be dropped because the collections containing the definition are already defined as lazy. Moreover in all other places where we have a `many-to-many`, the definition is done like this. So I don't think we are doing anything particular that could require extra laziness. To be extra sure however, I can have a deeper look and write some specific unit test.

Reply:
I removed this commit from the PR. It will be part of a different PR with additional testing.